src/contexts/Auth.tsx

```diff
@@ -0,0 +1,70 @@
+import React, {createContext, useState, useContext, useEffect} from 'react'
+import AsyncStorage from '@react-native-async-storage/async-storage';
+
+import {AuthData, authService} from '../services/authService';
+
+type AuthContextData = {
+    authData?: AuthData;
+    loading: boolean;
+    logIn(): Promise<void>;
+    logOut(): void;
+};
+
+const AuthContext = createContext<AuthContextData>({} as AuthContextData);
```

Comment:
According specification we can use Redux state managment library, why we need to use context?